openstack/input.yaml

```diff
@@ -9,46 +9,47 @@ dockerhub-pass:
 docker-public-registry-url: registry.hub.docker.com
 docker-registry-cert-country-name: SI
 docker-registry-cert-organization-name: XLAB
-docker-registry-cert-email-address: some_email@email.com
+docker-registry-cert-email-address: mail@xlab.si
 postgres_env:
-  POSTGRES_USER: username
-  POSTGRES_PASSWORD: password
+  POSTGRES_USER: username-blueprint-database
+  POSTGRES_PASSWORD: pass-blueprint-database
   POSTGRES_DB: postgres
 xopera_env:
   XOPERA_GIT_TYPE: gitlab
   XOPERA_GIT_URL: https://gitlab.com
-  XOPERA_GIT_AUTH_TOKEN: some_token
+  XOPERA_GIT_AUTH_TOKEN: gitlab-token
   XOPERA_VERBOSE_MODE: debug
   XOPERA_DATABASE_IP: xopera-postgres
   XOPERA_DATABASE_NAME: postgres
-  XOPERA_DATABASE_USER: username
-  XOPERA_DATABASE_PASSWORD: password
+  XOPERA_DATABASE_USER: username-blueprint-database
+  XOPERA_DATABASE_PASSWORD: pass-blueprint-database
   OIDC_INTROSPECTION_ENDPOINT: http://keycloak:8080/auth/realms/SODALITE/protocol/openid-connect/token/introspect 
-  OIDC_CLIENT_SECRET: a8337e6d-ffbf-45ac-9fef-a8d3bd5c45cb 
+  OIDC_CLIENT_SECRET: client-secret 
```

Comment:
Should we add that we expect (uuid) like input here?

Reply:
The same probably goes for all token inputs - just as a guideline for the user..

Reply:
This file should have been deleted by now, it slipped through despite .gitignore :D
​​​​​​​

Reply:
Maybe if we mention it in README#Input guidelines? We could also use regex to validate token-like inputs